.github/workflows/publish.yml

```diff
@@ -0,0 +1,49 @@
+name: Publish
+
+on:
+  schedule:
+    - cron: '0 0 1 */2 *'
+  workflow_dispatch:
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone the repository
+        uses: actions/checkout@v2
+      - name: Set up Python 3
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+      - name: Delete previously generated files
+        run: rm -rf megago/resources && mkdir -p megago/resources
+      - name: Download the most recent SwissProt associations file
+        run: wget "https://www.uniprot.org/uniprot/?query=reviewed:yes&format=tab&force=true&columns=id,go-id" -O megago/resources/associations-swissprot.tab
+      - name: Download the most recent GO DAG file
+        run: wget "http://purl.obolibrary.org/obo/go/go-basic.obo" -O megago/resources/go-basic.obo
+      - name: Install required dependencies
+        run: |
+          python3 -m pip install --upgrade pip
+          pip3 install -r requirements-dev.txt
+          pip3 install wheel
+      - name: Precompute the frequency counts
+        run: python3 -m megago.precompute_frequency_counts
+      - name: Precompute the highest IC values
+        run: python3 -m megago.precompute_highest_ic
+      - name: Install Twine (required for publishing the package on PyPi)
+        run: pip3 install twine
+      - name: Delete previously generated distributions
+        run: rm -rf dist
+      - name: Get UniProt database version
+        id: date
+        run: echo "::set-output name=date::$(wget -O - https://www.uniprot.org/downloads 2> /dev/null | grep 'uniprot.releasedate' |sed 's/^.* .\([_0-9]*\).*$/\1/' | sed 's/_/./')"
+      - name: Setup correct version numbering
+        run: sed -i "s/version='\([^']*\)'/version='\1.${{ steps.date.outputs.date }}'/" setup.py
+      - name: Generate packages for distribution
+        run: python setup.py sdist bdist_wheel
+      - name: Upload generated packages to PyPi
+        run: twine upload dist/*
+        env:
+          TWINE_USERNAME: ${{ secrets.TWINE_USERNAME }}
+          TWINE_PASSWORD: ${{ secrets.TWINE_PASSWORD }}
+
```

Comment:
Awesome! I will hopefully remember this action if I ever want to do something similar.

Reply:
One more remark: how about restricting the publishing to the master branch? Simply to avoid publishing feature/bugfix-branches by accident.

Reply:
I would rather disable automatic publishing all together (except for the automatic bi-monthly release). That way we can decide to bundle multiple PR's and release them at once

Reply:
I absolutely agree, but right now it's possible to trigger the action on branches, isn't it?

Reply:
Yes, it is :) This was merely something I added to test the publishing feature, but I'll remove it together with the other changes I need to perform to this PR :) Should be done by tonight! I'll re-request your review once this has been done